moduleroot/.github/workflows/cron.yml.erb

```diff
@@ -0,0 +1,94 @@
+name: Nightly tests
+
+on:
+  schedule:
+    - cron: '4 * * * *'
+
+jobs:
+  unit:
+    if: github.repository == '<%= @configs[:namespace] %>/<%= @configs[:puppet_module] %>'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.5"
+          - "2.4"
+        puppet:
+          - "6"
+          - "5"
+        exclude:
+          - ruby: "2.5"
+            puppet: "5"
+          - ruby: "2.4"
+            puppet: "6"
+    name: Puppet ${{ matrix.puppet }} (Ruby ${{ matrix.ruby }})
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-${{ matrix.puppet }}-${{ matrix.ruby }}-gems-unit-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ matrix.puppet }}-${{ matrix.ruby }}-gems-unit-
+      - name: Install dependencies
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle config without 'development system_tests'
+          bundle install --jobs 4 --retry 3
+        env:
+          PUPPET_VERSION: "${{ matrix.puppet }}.0"
```

Comment:
I think you can set `env` on the whole job, so you don't duplicate it in "run tests".

Reply:
Neat, will try.